src/test/java/app/sami/languageWeb/language/LanguageControllerTests.java

```diff
@@ -0,0 +1,189 @@
+package app.sami.languageWeb.language;
+import app.sami.languageWeb.auth.services.JwtService;
+import app.sami.languageWeb.language.dtos.LanguageGradeRequest;
+import app.sami.languageWeb.language.models.Language;
+import app.sami.languageWeb.language.models.LanguageGrades;
+import app.sami.languageWeb.testUtils.IntegrationTests;
+import app.sami.languageWeb.testUtils.factories.LanguageGradesFactory;
+import app.sami.languageWeb.testUtils.factories.UserFactory;
+import app.sami.languageWeb.user.models.User;
+import app.sami.languageWeb.user.repos.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+public class LanguageControllerTests extends IntegrationTests {
+
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    LanguageGradesRepository languageGradesRepository;
+    @Autowired
+    private JwtService jwtService;
+    private User userTest1;
+    private User userTest2;
+    private User userTest3;
+    private User userEvaluator;
+    private User userEvaluator2;
+    private LanguageGrades languageGradesTest1;
+    private LanguageGrades languageGradesTest2;
+    private Double grade1 = 50.0;
+    private Double grade2 = 100.0;
+    @BeforeEach
+    void setup(){
+        languageGradesRepository.deleteAll();
```

Comment:
no need to `deleteAll` since you added `@Transactional` on top of `IntegrationTests`